README_DEVELOPERS.md

```diff
@@ -1,17 +1,87 @@
 # kuberpult Readme for developers
 
-## Release a new version
+## introduction
 
-Building with libgit2 is tricky atm. Run `./dmake make -C services/cd-service bin/main` once to generate the binary for the cd-service.
-Afterwards run `make release`. This will push the docker image, package the helm chart and create a git tag. The helm chart must be uploaded manually to the github release at the moment.
-Afterwards bump the version in the `version` file.
+kuberpult watches a remote repository, unlike ArgoCD it is not triggered based on push to the repository, It is triggered by rest api instead (or ui which in turn calls the rest api). 
```

Comment:
kuberpult doesnt watch the repository at all. Instead it uses git as data storage. You also would say that an application watches MySQL or Cassandra.